lua/tree.lua

```diff
@@ -28,6 +29,21 @@ function M.open()
   end
 end
 
+
+local function go_to_git_item(mode)
+  local icon_state = config.get_icon_state()
+  if not icon_state.show_git_icon then return end
+
+  local icons = {}
+  for _, icon in pairs(icon_state.icons.git_icons) do
+    table.insert(icons, icon)
+  end
+
+  icons = table.concat(icons, '\\|')
+  local flags = mode == 'prev_git_item' and 'b' or ''
```

Comment:
you could make a closure over this variable and load specific functions for each action.
```lua
function gen_go_to(mode)
   local flags = mode == 'prev_git_item' and 'b' or ''
   return function()
    -- uses mode
   end
end

local go_to_prev = gen_go_to('prev_git_item')
local go_to_next = go_to_to()
```

Reply:
Done, thanks.